README.md

```diff
@@ -17,8 +17,66 @@ Implemented:
 - translation hooks
 - enter string hooks (search)
 
-### build
+### build and run
+
+#### install rustc, toolchain by using rustup
+
+For install RUST and Cargo [see link](https://doc.rust-lang.org/cargo/getting-started/installation.h).
+
+Or type:
+```shell
+curl https://sh.rustup.rs -sSf | sh
+```
+
+NOTE: The use of system rustc and cargo often leads to build errors.
+
+Example:
+```
+error: no such command: `+nightly-2022-11-06`
+
+  Cargo does not handle `+toolchain` directives.
+  Did you mean to invoke `cargo` through `rustup` instead?
+
+```
+
+After loading $HOME/.cargo/env:
+
+```
+source "$HOME/.cargo/env"
+```
+
+Install nightly-2022-11-06 toolchain from rustup:
+
+```shell
+rustup install nightly-2022-11-06-x86_64-unknown-linux-gnu
+
+```
+
+#### build
```

Comment:
In general, it's better to make the first letter of a header capital, and don't use h4 and smaller headers.